src/cimsparql/model.py

```diff
@@ -544,8 +544,12 @@ def st_group_codes_names_query(self) -> str:
 
     @time_it
     def station_group_codes_and_names(self) -> StationGroupCodeNameDataFrame:
-        df = self.get_table_and_convert(self.st_group_codes_names_query)
-        return StationGroupCodeNameDataFrame(df.set_index("station_group"))
+        df = self.get_table_and_convert(self.st_group_codes_names_query).set_index("station_group")
+        if df.index.has_duplicates:
+            duplicate_indices = df.index[df.index.duplicated(keep="first")]
+            logger.warning("Found duplicated names for following station groups: %s", df.loc[duplicate_indices])
```

Comment:
Det er viktig å huske her at det er ikke en stasjonsgruppe som har to navn (som jeg synes denne meldingen indikerer). To navn på en gruppe, ville ikke vært noe problem. Det er to ulike stasjonsgrupper som har samme markedskode (og markedskoden skal være unik). Det er litt tilfeldig at det var denne sjekken som "slo" ut. Så hvis det ligger bud på den markedskoden så må man forvente at allokeringen blir rimelig feil. Kan godt tenkes at det kommer et bud som var ment for den gruppe A, men som i sin helthet blir plassert på gruppe B. Det er jo teknisk mulig at man greier å legge inn to navn på en gruppe også (men det vil jo bryte CIM så der tenker jeg det er mer validering). Det at markedskoden ikke er unik bryter SN-extension til CIM15 (og det ser ut til at det er dårlig validering på SN-ext). @vegraux kan man hvertfall legge til `or there are multiple station groups with the same market code`?